editor/import/resource_importer_texture_atlas.cpp

```diff
@@ -126,7 +134,7 @@ static void _plot_triangle(Vector2i *vertices, const Vector2i &p_offset, bool p_
 	double dx_low = double(x[2] - x[1]) / (y[2] - y[1] + 1);
 	double xf = x[0];
 	double xt = x[0] + dx_upper; // if y[0] == y[1], special case
-	int max_y = MIN(y[2], height - p_offset.y - 1);
+	int max_y = y[2];
```

Comment:
Would you mind explaining the reasoning for this change?

Reply:
The check was faulty, at least in that form, might be that it should instead be based on the transposed status will check

Reply:
Changing it to take transposed into account resolves the problem, adding that